sonar-db/src/main/java/org/sonar/db/user/UserTokenDao.java

```diff
@@ -58,18 +56,17 @@ public UserTokenDto selectOrFailByTokenHash(DbSession dbSession, String tokenHas
     return mapper(dbSession).selectByLogin(login);
   }
 
-  public Map<String, Integer> countTokensByLogins(final DbSession dbSession, List<String> logins) {
-    final Map<String, Integer> result = new HashMap<>();
-    DatabaseUtils.executeLargeInputs(logins, new Function<List<String>, List<UserTokenCount>>() {
-      @Override
-      public List<UserTokenCount> apply(@Nonnull List<String> input) {
-        List<UserTokenCount> userTokenCounts = mapper(dbSession).countTokensByLogins(input);
-        for (UserTokenCount userTokenCount : userTokenCounts) {
-          result.put(userTokenCount.getLogin(), userTokenCount.tokenCount());
-        }
-        return userTokenCounts;
-      }
-    });
+  public Map<String, Integer> countTokensByLogins(DbSession dbSession, List<String> logins) {
+    Map<String, Integer> result = new HashMap<>(logins.size());
+    executeLargeInputs(
+        logins,
+        input -> {
+          List<UserTokenCount> userTokenCounts = mapper(dbSession).countTokensByLogins(input);
+          for (UserTokenCount userTokenCount : userTokenCounts) {
+            result.put(userTokenCount.getLogin(), userTokenCount.tokenCount());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 9. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)